src/handlers/motions/motionFinalized/helpers.ts

```diff
@@ -49,3 +52,48 @@ export const getStakerReward = async (
     isClaimed: false,
   };
 };
+
+export const getParsedActionFromMotion = async (
```

Comment:
Maybe we call this `getParsedActionFromDomainMotion` to distinguish between the more generic function of the same name in the MotionCreated helpers?